srcalc/src/main/webapp/WEB-INF/views/confirmNewCalc.jsp

```diff
@@ -0,0 +1,16 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib uri="/WEB-INF/srcalc.tld" prefix="srcalc" %>
+
+<srcalc:calcPage title="Confirm New Calculation">
+<section>
+    <h2>Confirm New Calculation</h2>
+    Calculation in progress for ${calculation.patient.name}. Starting a new calculation will overwrite the in-progress calculation.<br>
```

Comment:
Putting line breaks here isn't really good English. How about splitting it into proper paragraphs? You could have one paragraph for explaining the circumstance and another (or even a bullet list) for the available actions.